solutions/gatekeeper-policies/guardrails/Kptfile

```diff
@@ -0,0 +1,8 @@
+apiVersion: kpt.dev/v1
+kind: Kptfile
+metadata:
+  name: guardrails-policies
+  annotations:
+    config.kubernetes.io/local-config: "true"
+info:
+  description: sample description
```

Comment:
Is this file more of a template? If not, perhaps the description needs to be more specific?

Reply:
@lucstjean-ssc description updated. please mark as resolved if you like it